protobuf-kafkaconnect-converter/src/main/java/com/amazonaws/services/schemaregistry/kafkaconnect/protobuf/fromconnectschema/ProtobufSchemaConverterUtils.java

```diff
@@ -15,4 +17,32 @@ public static String toMapEntryName(String s) {
         s = s.substring(0, 1).toUpperCase() + s.substring(1);
         return s;
     }
+
+    public static java.util.Date convertFromGoogleDate(com.google.type.Date date) {
+        TimeZone timeZone = TimeZone.getTimeZone("UTC");
+        Calendar cal = Calendar.getInstance(timeZone);
+        cal.setLenient(false);
+        cal.set(Calendar.YEAR, date.getYear());
+        cal.set(Calendar.MONTH, date.getMonth() - 1); // Months start at 0, not 1
+        cal.set(Calendar.DAY_OF_MONTH, date.getDay());
+        cal.set(Calendar.HOUR_OF_DAY, 0);
+        cal.set(Calendar.MINUTE, 0);
+        cal.set(Calendar.SECOND, 0);
+        cal.set(Calendar.MILLISECOND, 0);
+        return cal.getTime();
+    }
+
+    public static java.util.Date convertFromGoogleTime(com.google.type.TimeOfDay time) {
+        TimeZone timeZone = TimeZone.getTimeZone("UTC");
+        Calendar cal = Calendar.getInstance(timeZone);
+        cal.set(Calendar.YEAR, 1969); //year, month, and day must be hardcoded to match
```

Comment:
answered in the comment on line 38 - java.util.Date has a hardcoded year as a default. Since we are using it as the format to convert the time, which doesn't have a date, and we cannot set the date to 0 due to typing issues, I hardcoded the date so it will match in the conversion. We can talk in more detail later if you have a better idea

Reply:
Can you add some references in the comment? For example documentation for `java.util.Date`

Reply:
Looked into it - the date is actually for the universal unix epoch. added reference in comments

Reply:
Thanks for looking into this! Can you change it into one single comment in the front instead of two comments?  You might want to reference https://en.wikipedia.org/wiki/Unix_time and https://docs.oracle.com/javase/8/docs/api/java/time/Instant.html for more clearance. Also, just curious why we cant use 1970-01-01 according to the documentation instead of this 1969-11-32?